src/components/header/EmptyThreadIndicator.css

```diff
@@ -0,0 +1,36 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+.headerEmptyThreadIndicator {
+  --empty-buffer-color1: rgba(69, 160, 255, 0.3);
+  --empty-buffer-color2: rgba(69, 160, 255, 0.1);
+}
+
+.headerEmptyThreadIndicatorBlock {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+}
+
+.headerEmptyThreadIndicatorStartup {
+  border-right: solid 1px var(--grey-30);
+  background-color: var(--grey-20);
+  box-shadow: inset -4px 0 4px rgba(0, 0, 0, 0.05);
+}
+
+.headerEmptyThreadIndicatorShutdown {
+  border-left: solid 1px var(--grey-30);
+  background-color: var(--grey-20);
+  box-shadow: inset 4px 0 4px rgba(0, 0, 0, 0.05);
+}
```

Comment:
These indicators hide the vertical time lines. I find it confusing. A good solution is to add an opacity to these two indicators. From my test "0.5" works fine for me.